src/UserFieldsCreateTemplate.php

```diff
@@ -25,17 +25,14 @@ function addInputItem( $name, $value, $type, $msg ) {
 
 	function execute() {
 		global $sfgTabIndex;
-
-		if ( !$this->haveData( 'header' ) ) {
-			return '';
-		}
+		global $wgOut; /* No other way to access http://www.mediawiki.org/wiki/Manual:$wgOut */
+		$wgOut->addModules( 'ext.semanticforms.main' );
 
```

Comment:
I was expecting `QuickTemplate` to have access to `getSkin` because of https://github.com/wikimedia/mediawiki/blob/073d4001f5bd1ac7d52c1a34d534ff3a59c2863f/includes/skins/QuickTemplate.php#L177